### ISSUE TITLE
[Feature]: Show link to extension settings page after running `app deploy`

### ISSUE THREAD

**Issue by Mtillmann** [Shopify/cli#1205]
### What area(s) will this request affect?

Extension

### What type of change do you want to see?

New feature

### Overview

CLI 2.0 would show a link to the page where you can create a version and publish your recently deployed/pushed changes. I think it'd be a good idea to bring this back because it was really handy

### Motivation

I have to navigate through the partner site to find the page manually. With CLI 2.0 it was a  lot easier

**Comment by matteodepalo** [comment]
@Mtillmann thank you for the feedback, I'll bring it up with the team.

**Comment by stephszeto** [comment]
Hm, not sure if I'm missing something, but there should be a link to the extension(s) deployed that are included in the success banner. What does your success banner look like, Martin?

<img width="853" alt="Screenshot 2023-01-26 at 2 06 02 PM" src="https://user-images.githubusercontent.com/71412066/214961246-4e1c6f1d-bdf1-4ab6-b980-869d14556061.png">

**Comment by Mtillmann** [comment]
Yes, it's a theme app extension. I've modified my npm script to dump the url after deployment so it's no big deal but I was a bit confused because the old Cli app always shows the url upon deploy.

**Comment by Mtillmann** [comment]
Oh and the version I'm using is the one in my Screenshot

**Comment by matteodepalo** [comment]
I've found the source of the problem, we don't show the link if there are only theme extensions. I'll fix it and close this once it's released.
